src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
         Console.Write(rn);
     }
 }";
-            CompileAndVerify(source: source, expectedOutput: "TwoZero");
+            CompileAndVerify(source: source, expectedOutput: "TwoZero")
+                .VerifyOperationTree(@"
+Test.Main()
```

Comment:
I like the approach as well. Instead of having the operation tree inline though, can we instead have a partial type with constants like `TestBug11947_OperationTree` and the tests just get an extra line `.VerifyOperationTree(TestBug11947_OperationTree)` so that it doesn't clutter the binding tests?